Formula/lilypond.rb

```diff
@@ -29,8 +29,6 @@ class Lilypond < Formula
 
   head do
     url "https://gitlab.com/lilypond/lilypond.git", branch: "master"
-    mirror "https://github.com/lilypond/lilypond.git"
-    mirror "https://git.savannah.gnu.org/git/lilypond.git"
```

Comment:
While adding the default branch, it caused some syntax issue

```
  Error: lilypond: wrong number of arguments (given 2, expected 1)
```

https://github.com/Homebrew/homebrew-core/actions/runs/3819450686/jobs/6497020854